src/test/java/org/example/tms/exception/ValidationExceptionMessageBuilderTest.java

```diff
@@ -0,0 +1,86 @@
+package org.example.tms.exception;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.validation.FieldError;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ValidationExceptionMessageBuilderTest {
```

Comment:
perfect! nice work! I see tests but dont have time to check them, @Caromeen  u can check them if u have free time)